### PR TITLE
Improve remove link

### PR DESCRIPTION
**Pull request by davidglezz**
- Use icon instead uppercase X.
- Add title for accessibility.

### CODE REVIEW

File: views/templates/front/mailalerts-account-line.tpl [hunk 1]
```diff
@@ -28,9 +28,10 @@
   {$mailAlert.name}
   <span>{$mailAlert.attributes_small}</span>
   <a href="#"
-     class="js-remove-email-alert"
+     title="{l s='Remove mail alert' d='Modules.Mailalerts.Shop'}"
+     class="js-remove-email-alert btn btn-link"
      rel="js-id-emailalerts-{$mailAlert.id_product|intval}-{$mailAlert.id_product_attribute|intval}"
      data-url="{url entity='module' name='ps_emailalerts' controller='actions' params=['process' => 'remove']}">
-    X
+    <i class="material-icons">delete</i>
```
Comment (PierreRambaud) [inline views/templates/front/mailalerts-account-line.tpl]: Are you sure material is available in PS 1.7.1? https://github.com/PrestaShop/ps_emailalerts/blob/master/ps_emailalerts.php#L68

Comment (davidglezz) [reply views/templates/front/mailalerts-account-line.tpl]: Sorry, I don't know, if it isn't, we can change it to &times; (×).
You know it?

Comment (davidglezz) [reply views/templates/front/mailalerts-account-line.tpl]: Yes

https://github.com/PrestaShop/PrestaShop/blob/b9805532d49594e40765542345936086c5b8441f/themes/classic/_dev/package.json#L23

https://github.com/PrestaShop/PrestaShop/blob/1.7.1.x/themes/classic/assets/css/theme.css

And search "Material Icons"

Comment (PierreRambaud) [reply views/templates/front/mailalerts-account-line.tpl]: Thanks :+1:

**Comment by Robin-Fischer-PS** [comment]
Hi @davidglezz ! Thanks for the PR.
Could you please give us a scenario for testing this PR ? :)

**Comment by davidglezz** [comment]
I'll try:
- You have to install the module.
- Activate the hook displayProductAdditionalInfo.
- Set a product without stock.
- With the session initiated as a client, go to the product page and press the module button "notify me when available" or similar text.
- Go to "my account" click on the mail alerts icon.
- The previous product appears in the list.
- The last X of the name is to remove the notification.
- I have changed the X for an icon and improvements in accessibility.

**Comment by Robin-Fischer-PS** [comment]
Thanks @davidglezz for the PR and the scenario, it's OK for me :)

**Pull request merged**